src/firefly/js/tables/ui/BasicTable.jsx

```diff
@@ -30,21 +28,18 @@ export class BasicTable extends React.Component {
             columnWidths: makeColWidth(props.columns, props.data, props.showUnits)
         };
 
-        this.onResize = this.onResize.bind(this);
+        this.onResize =  debounce((size) => {
+                if (size) {
+                    var widthPx = size.width;
+                    var heightPx = size.height;
+                    this.setState({widthPx, heightPx});
+                }
+            }, 100);
```

Comment:
Yep, the debounce was on purpose.  I wanted it to be more responsive.

The small table not rendering is related to the 'scrollToRow bug' I mentioned as fixed.  For some reason, the fix was not in.  I added it now.

I agree with the selected tab comment.  The current behavior is temporary.  It will mimic that of the old application once converted.